portfolio/src/main/webapp/script.js

```diff
@@ -452,6 +458,78 @@ function logoutHandler(value){
     //removeClass("chart-container", "blur-content"); not necessary for this pr
 }
 
+//*********************** VOTING HANDLING ***********************
+/**
+ * Listens for the selected genre by the user
+ */ 
+function listenForGenreChoice(){
+    const form = document.getElementById("chartForm");
+    const el = document.getElementsByName('genre'); 
```

Comment:
use `document.querySelector('[name="genre"]:checked')` to get the currently selected radio button.